modules/components/pages/inputs/kafka_migrator.adoc

```diff
@@ -0,0 +1,708 @@
+= kafka_migrator
+// tag::single-source[]
+:type: input
+:status: beta
+:categories: ["Services"]
+
+// © 2024 Redpanda Data Inc.
+
+
+component_type_dropdown::[]
+
+Use this connector in conjunction with the xref:components:outputs/kafka_migrator.adoc[`kafka_migrator` output] to migrate topics between Kafka brokers. The `kafka_migrator` input uses the https://github.com/twmb/franz-go[Franz Kafka client library^].
+
+
+ifndef::env-cloud[]
+Introduced in version 4.35.0.
+endif::[]
+
+[tabs]
+======
+Common::
++
+--
+
+```yml
+# Common configuration fields, showing default values
+input:
+  label: ""
+  kafka_migrator:
+    seed_brokers: [] # No default (required)
+    topics: [] # No default (required)
+    regexp_topics: false
+    consumer_group: "" # No default (optional)
+    auto_replay_nacks: true
+```
+
+--
+Advanced::
++
+--
+
+```yml
+# All configuration fields, showing default values
+input:
+  label: ""
+  kafka_migrator:
+    seed_brokers: [] # No default (required)
+    topics: [] # No default (required)
+    regexp_topics: false
+    consumer_group: "" # No default (optional)
+    client_id: benthos
+    rack_id: ""
+    batch_size: 1024
+    auto_replay_nacks: true
+    commit_period: 5s
+    start_from_oldest: true
+    tls:
+      enabled: false
+      skip_cert_verify: false
+      enable_renegotiation: false
+      root_cas: ""
+      root_cas_file: ""
+      client_certs: []
+    sasl: [] # No default (optional)
+    multi_header: false
+    batching:
+      count: 0
+      byte_size: 0
+      period: ""
+      check: ""
+      processors: [] # No default (optional)
+    topic_lag_refresh_period: 5s
+    output_resource: kafka_migrator_output
+```
+
+--
+======
+
+The `kafka_migrator` input:
```

Comment:
@Jeffail. Please can you check this intro carefully to make sure the sequence is correct?